lbry/lbry/extras/daemon/Components.py

```diff
@@ -112,6 +108,7 @@ def __init__(self, component_manager):
         self.headers_dir = os.path.join(self.conf.wallet_dir, 'lbc_mainnet')
         self.headers_file = os.path.join(self.headers_dir, 'headers')
         self.old_file = os.path.join(self.conf.wallet_dir, 'blockchain_headers')
+        self.headers = Headers(self.headers_file)
```

Comment:
Is this different from `wallet_manager.ledger.headers`? In `get_status()` it uses the `ledger.headers` object but in other places in this component it uses this new `self.headers` object, is that correct?

Reply:
`get_status` usage of wallet `ledger.headers` is only to keep reporting status as the wallet downloads from the server. At this point (when this component itself stops and wallet starts) there is no more self.headers, so it forwards status calls.
This is just to keep the API the same and not break the app while reporting wallet progress after a failed HTTP download. When the wallet is able to efficiently download from server we can drop this entire component, which I think should be soon.